lib/core/utils/preload-cssom.js

```diff
@@ -32,7 +32,7 @@ axe.utils.preloadCssom = function preloadCssom({
 		return q;
 	}
 
-	const dynamicDoc = document.implementation.createHTMLDocument();
+	const dynamicDoc = document.implementation.createHTMLDocument('New Document');
```

Comment:
I thought @jkodu had already fixed this? 🤔

Reply:
He probably did, I probably need to rebase on develop again.

Reply:
Actually, looks like he has it for the test, but not for the util. 

https://github.com/dequelabs/axe-core/blob/08e4447d636e98fe2e7424ec856bc1b56ccaec94/test/checks/mobile/css-orientation-lock.js#L6-L8

Reply:
I do have it for the util too - https://github.com/dequelabs/axe-core/pull/1339/files#diff-93d57bbe6c6d128e06d4fe309e06ff40R34

The PR is not merged yet.